crates/vfs/src/vfs_path.rs

```diff
@@ -292,7 +292,7 @@ impl From<AbsPathBuf> for VfsPath {
 impl fmt::Display for VfsPath {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match &self.0 {
-            VfsPathRepr::PathBuf(it) => fmt::Display::fmt(&it.display(), f),
+            VfsPathRepr::PathBuf(it) => fmt::Debug::fmt(&it, f),
             VfsPathRepr::VirtualPath(VirtualPath(it)) => fmt::Display::fmt(it, f),
         }
     }
```

Comment:
Is this intentional? I think `fmt::Display::fmt(&it, f)` is correct here.

Reply:
Argh, good catch, this is intentional, but messed up!

In *this* impl, we want to use Display, in the one below we want to use debug.

that is,

```
fmt::Debug::fmt(&it.display(),
```

was actually wrong (this calls display impl, but we do want debug here), but I messed this up.

Could you PR this? :)